### DISCUSSION TITLE
View the count of achievements cards received between specific numbers.

### DISCUSSION THREAD

**Discussion by ashwin3454** [masterking32/MasterHamsterKombatBot#402] [Ideas]
eg: May be i've watched 60 videos. but it does not showing in hamster.
is it possible to view the real count in the script?

![-2147483648_-210390](https://github.com/user-attachments/assets/0348d25e-21b5-40bb-9ff5-11d88303e9f1)

**Comment by Fy0urM** [comment]
server response with user's achievements does not contain information about the number of performed actions
![image](https://github.com/user-attachments/assets/3ce3a06d-3bc6-4d3e-90ab-4f06f57d93c4)
the config request response contains only the target values for each achievement
![image](https://github.com/user-attachments/assets/c371f16c-c6a3-4b3d-9568-dee0d1ffd6a1)
It's probably counted on the backend

**Reply by ashwin3454** [reply]
okay
